test/integration/integration_test.go

```diff
@@ -84,15 +84,15 @@ func TestMain(m *testing.M) {
 		godogTags += "~cdk-only"
 		isoUrl := os.Getenv("MINISHIFT_ISO_URL")
 		switch isoUrl {
-		case "b2d":
+		case "", "b2d":
```

Comment:
:+1:

Reply:
`00:06:56.424 Test run using Boot2Docker iso image.`

Test are running with b2d iso at the moment. Could you update the code to somehow pass minikube iso ? or `minikube` in case of empty `isoUrl` ?

Reply:
@budhrg So, are we switching for minikube.iso as default? If so I can then switch it and we will have no work with setting up env variable in centos.sh. Let me know.